code/controllers/subsystem/ticker.dm

```diff
@@ -479,6 +479,8 @@ SUBSYSTEM_DEF(ticker)
 			livings += living
 			if(living.client && length(living.client?.active_challenges))
 				SSchallenges.apply_challenges(living.client)
+			if(living.mind.assigned_role.departments_bitflags & DEPARTMENT_BITFLAG_SECURITY)
```

Comment:
convert this into a variable in Ticker so we can create a dynamic need for users to join jobs based on demands